### ISSUE TITLE
add support for jackd

### ISSUE THREAD

**Issue by alekibango** [wahjam/wahjam#65]
I am a guitar player and i need my stack of audio tools, best of which is http://rakarrack.sf.net
sometimes i also use  http://www.hydrogen-music.org or http://ardour.org/ 

All those tools (and many more) can be connected (online) using jackd.  
This worked well for me with an unofficial version of curses ninjam client with jackd support enabled. 

Wahjam could be really cool if it would support jackd...  Please add support for jackd! Without that it is not possible to use great number of sound tools ...

**Comment by federicobriata** [comment]
Hi, I got too problem using jack2 with wahjam.
anybody have some tips for make it work?

**Comment by stefanha** [comment]
@federicobriata I am using jack-audio-connection-kit-1.9.12-10.fc31.x86_64 on Fedora 31.  Have you tried jack 1.9 instead of jack2?

**Comment by federicobriata** [comment]
@stefanha Jack2 it is 1.9, I'm using 1.9.12 too on my Debian 10, but I made the test on April 6th.
I see now that in meantime you made some new commit, could it explain why on your side now is working?

**Comment by stefanha** [comment]
@federicobriata You are right, the jack-audio-connection-kit package on Fedora is built from jack2 sources.

The commit I merged this morning (https://github.com/wahjam/wahjam/commit/2b3ee086bc8c35a01a3efddbb32708171e32d6bb) fixes a crash when audio starts.  It's not unrelated to JACK and also happens with ALSA.  If Wahjam crashes when starting audio and the backtrace shows EffectProcessor::process() then it's probably the same bug and the commit will solve the issue for you.

**Comment by federicobriata** [comment]
yes, that was the issue I saw! So 2b3ee08 solve it.
thanks a lot and happy easter! :)

**Comment by stefanha** [comment]
@federicobriata Glad to hear!  Have a happy easter too!
